go.mod

```diff
@@ -1,6 +1,7 @@
 module fortio.org/fortiotel
 
 go 1.21
+toolchain go1.22.5
```

Comment:
this should not be needed

also not sure why otel thinks it's a good idea to force upgrade to 1.22